nautobot/core/templates/panel_table.html

```diff
@@ -13,4 +14,16 @@
     {% else %}
         <div class="panel-body text-muted">None</div>
     {% endif %}
+    {% if add_obj_url %}
+        <div class="panel-footer noprint">
+            <div class="pull-right">
+                <a href="{{ add_obj_url }}" class="btn btn-primary btn-xs">
+                    <span class="mdi mdi-plus-thick" aria-hidden="true"></span> Add {{ table.data.data.model|meta:"verbose_name_plural" }}
```

Comment:
Perhaps just `verbose_name`? Since normally this button lets you just add a single record.